rust/Cargo.toml.in

```diff
@@ -72,3 +72,10 @@ suricata-lua-sys = { git = "https://github.com/jasonish/suricata-lua-sys", versi
 [dev-dependencies]
 test-case = "~3.3.1"
 hex = "~0.4.3"
+
+[patch.crates-io]
+# Use custom branch of time as 0.3.20 is required for our MSRV of
+# 1.63, but 0.3.20 also fails to build on latest Rust.
+#
+# With Rust 1.67+ we can move to time 0.3.35 and not worry about this.
+time = { version = "=0.3.20", git = "https://github.com/jasonish/time", branch = "0.3.20-suricata" }
```

Comment:
This is a branch, but if we want to go this route, I'd give it a proper tag. Would we also want to drop it under the OISF account?

Reply:
Not really happy with this, but I guess we may have to do something like it. If we go this route, would it make sense to take a newer release and patch it to build on 1.63?

Reply:
> Not really happy with this, but I guess we may have to do something like it. If we go this route, would it make sense to take a newer release and patch it to build on 1.63?

No, newer time has dependencies that require 1.67 - it would be a rabbit hole.

Any ideas for other ways to handle this?

Reply:
Package renaming like we have done with Nom for 2 different versions can't be used here, as these 2 versions of time of "too close" in terms of sem-ver: https://redmine.openinfosecfoundation.org/issues/7130#note-5

Reply:
Another solution: `git submodule`s perhaps as they're are often used to solve version inconsistencies by maintaining a specific pinned version of a lib.

Reply:
If we're maintaining a fork anyway, I'd rather use cargo's logic for using it.